.github/workflows/docs.yml

```
@@ -22,7 +22,7 @@ jobs:
 
       - name: Load environment file
         id: dotenv
-        uses: falti/dotenv-action@v1
+        uses: falti/dotenv-action@v1.1
```

Please keep using v1 so the version does not need to be manually updated every time the action gets updated.
```suggestion
        uses: falti/dotenv-action@v1
```

---

Thanks, applied with Co-authored-by!